### ISSUE TITLE
Pagination doesn't work in postgres

### ISSUE THREAD

**Issue by Ezhvsalate** [uriyyo/fastapi-pagination#1]
Trying to follow your example with databases (https://github.com/uriyyo/fastapi-pagination/blob/main/examples/pagination_databases.py) I failed with following error:

```
  File "c:\git\routemaker\venv\lib\site-packages\fastapi_pagination\ext\databases.py", line 13, in paginate
    total = await db.fetch_val(select([func.count()]).select_from(query))
  File "c:\git\routemaker\venv\lib\site-packages\databases\core.py", line 155, in fetch_val
    return await connection.fetch_val(query, values, column=column)
  File "c:\git\routemaker\venv\lib\site-packages\databases\core.py", line 256, in fetch_val
    return await self._connection.fetch_val(built_query, column)
  File "c:\git\routemaker\venv\lib\site-packages\databases\backends\postgres.py", line 188, in fetch_val
    row = await self.fetch_one(query)
  File "c:\git\routemaker\venv\lib\site-packages\databases\backends\postgres.py", line 168, in fetch_one
    row = await self._connection.fetchrow(query, *args)
  File "c:\git\routemaker\venv\lib\site-packages\asyncpg\connection.py", line 477, in fetchrow
    data = await self._execute(query, args, 1, timeout)
  File "c:\git\routemaker\venv\lib\site-packages\asyncpg\connection.py", line 1446, in _execute
    query, args, limit, timeout, return_status=return_status)
  File "c:\git\routemaker\venv\lib\site-packages\asyncpg\connection.py", line 1454, in __execute
    return await self._do_execute(query, executor, timeout)
  File "c:\git\routemaker\venv\lib\site-packages\asyncpg\connection.py", line 1466, in _do_execute
    stmt = await self._get_statement(query, None)
  File "c:\git\routemaker\venv\lib\site-packages\asyncpg\connection.py", line 351, in _get_statement
    statement = await self._protocol.prepare(stmt_name, query, timeout)
  File "asyncpg\protocol\protocol.pyx", line 163, in prepare
asyncpg.exceptions.PostgresSyntaxError: subquery in FROM must have an alias
HINT:  For example, FROM (SELECT ...) [AS] foo.

```

**Comment by uriyyo** [comment]
Thanks for submitting an issue, will try to investigate and fix it

**Comment by Ezhvsalate** [comment]
Changing line #13 in file fastapi_pagination\ext\databases.py to
` total = await db.fetch_val(select([func.count()]).select_from(query.alias("subquery")))` helped me.

**Comment by uriyyo** [comment]
Thanks for pointing to the problem, I have already found it🙂

Fix of this issue will be included in the `0.4.0` version (I hope the release will be today).

**Comment by uriyyo** [comment]
This issue is fixed in [0.4.0](https://github.com/uriyyo/fastapi-pagination/releases/tag/0.4.0) version.

I am closing this issue, feel free to reopen if there are still issues with Postgres integration.
